crypto/src/test/kotlin/web5/sdk/crypto/Secp256k1Test.kt

```diff
@@ -53,6 +55,34 @@ class Secp256k1Test {
     val base64UrlEncodedSig2 = Convert(sig2).toBase64Url(padding = false)
 
     assertEquals(base64UrlEncodedSig1, base64UrlEncodedSig2)
+  }
+
+  @Test
+  fun `pressure test signature verification`() {
+    // TODO: consider using the same private key
+    val privateKey = Secp256k1.generatePrivateKey()
+    val publicKey = Secp256k1.computePublicKey(privateKey)
+
+    fun generateRandomString(length: Int): String {
+      val allowedChars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789"
+      val random = Random()
+      return (1..length)
+        .map { allowedChars[random.nextInt(allowedChars.length)] }
+        .joinToString("")
+    }
+
+
+
+    repeat(10_000) {
+      val payload = generateRandomString(Random().nextInt(100))
 
+      try {
+        val sig1 = Secp256k1.sign(privateKey, payload.toByteArray())
+        Secp256k1.verify(publicKey, payload.toByteArray(), sig1)
+      } catch (e: SignatureException) {
+        println("($it) $e. Payload: $payload")
+        throw e
+      }
```

Comment:
Perhaps this would be simpler? 
```suggestion
    repeat(10_000) {
      val payload = Random.nextBytes(Random.nextInt(100))

      try {
        val sig1 = Secp256k1.sign(privateKey, payload)
        Secp256k1.verify(publicKey, payload, sig1)
      } catch (e: SignatureException) {
        val payloadString = Convert(payload).toBase64Url(false)
        println("($it) $e. Payload (base64url encoded): $payloadString")
        throw e
      }
```

Reply:
awesome suggestion.